src/performance-metrics/spec.js

```diff
@@ -4,10 +4,18 @@ const IGNORE = [
 ];
 
 describe('performance-metrics', () => {
-    it('Should include all performance metrics', () => {
+    it('Default should include all performance metrics', () => {
         const windowMetrics = Object.keys(window.performance.timing.constructor.prototype)
             .filter((item) => !IGNORE.includes(item));
 
-        expect(performanceMetrics).to.have.members(windowMetrics);
+        expect(performanceMetrics()).to.have.members(windowMetrics);
+    });
+
+    it('Should pass an empty array', () => {
+        expect(performanceMetrics([])).to.have.lengthOf(0);
+    });
+
+    it('Should ignore metrics that are not valid', () => {
+        expect(performanceMetrics(['domInteractive', 'loadEventEnd', 'another'])).to.have.lengthOf(2);
```

Comment:
Maybe also add a test that checks that the correct metrics actually stay (not just count, but also value).